lib/index.js

```diff
@@ -24,10 +24,10 @@ class Oppsy extends Events.EventEmitter {
           psup: ProcessMonitor.uptime,
           psmem: ProcessMonitor.memoryUsage,
           psdelay: ProcessMonitor.delay,
-          requests: this._networkMonitor.requests,
-          concurrents: this._networkMonitor.concurrents,
-          responseTimes: this._networkMonitor.responseTimes,
-          sockets: this._networkMonitor.sockets
+          requests: this._networkMonitor.requests.bind(this._networkMonitor),
```

Comment:
I'm not sure about how to solve this the best way. In the `Network` constructor? `this.requests = this.requests.bind(this)`? That doesn't really make sense because the only reason the this pointer is out of what is because the thing calling it is messing it up. Shouldn't the `this` pointer manipulation be done there instead?